### DISCUSSION TITLE
does gotenberg not able to process multiple request concurrently??

### DISCUSSION THREAD

**Discussion by vivekportpro** [gotenberg/gotenberg#897] [Q&A] [answered]
If i hit 6 request concurrently to gotenberg server for generate pdf, it is taking much time  for process last pdf request.
![image](https://github.com/gotenberg/gotenberg/assets/168623201/a5f82dac-949c-4c3a-8f25-f4b2cc8f3091)

if i hit last request alone then it taking only 2 second to complete.

![image](https://github.com/gotenberg/gotenberg/assets/168623201/2d6d500f-9f5d-4b25-a187-a50cf842e5bd)

does gotenberg not able to process multiple request concurrently ?? or gotenberg process request in queue ??

**Comment by gulien** [comment, marked as answer]
Both Chromium and LibreOffice process the requests one by one. If you want concurrency, you should add more Gotenberg instances.

**Reply by gulien** [reply]
See also: https://github.com/gotenberg/gotenberg/discussions/893#discussioncomment-9689124
